scrapy/utils/project.py

```diff
@@ -68,18 +68,16 @@ def get_project_settings():
     if settings_module_path:
         settings.setmodule(settings_module_path, priority='project')
 
-    # XXX: remove this hack
     pickled_settings = os.environ.get("SCRAPY_PICKLED_SETTINGS_TO_OVERRIDE")
     if pickled_settings:
         warnings.warn("Use of environment variable "
                       "'SCRAPY_PICKLED_SETTINGS_TO_OVERRIDE' "
                       "is deprecated.", ScrapyDeprecationWarning)
-        settings.setdict(pickle.loads(pickled_settings), priority='project')
+    settings.setdict(pickle.loads(pickled_settings), priority='project')
```

Comment:
I’m guessing this indentation change was accidental :)

Reply:
Indeed, that's what making the build fail.